collectors/python.d.plugin/openldap/openldap.chart.py

```diff
@@ -0,0 +1,198 @@
+        ]
+    },
+    'entries_sent': {
+        'options': [None, 'Entries', 'entries/s', 'ldap', 'openldap.entries', 'line'],
+        'lines': [
+            ['entries_sent', 'sent', 'incremental']
+        ]
+    },
+    'ldap_operations': {
+        'options': [None, 'Operations', 'ops/s', 'ldap', 'openldap.ldap_operations', 'line'],
+        'lines': [
+            ['bind_operations', 'bind', 'incremental'],
+            ['search_operations', 'search', 'incremental'],
+            ['unbind_operations', 'unbind', 'incremental'],
+            ['add_operations', 'add', 'incremental'],
+            ['delete_operations', 'delete', 'incremental'],
+            ['modify_operations', 'modify', 'incremental'],
+            ['compare_operations', 'compare', 'incremental']
+        ]
+    },
+    'waiters': {
+        'options': [None, 'Waiters', 'waiters/s', 'ldap', 'openldap.waiters', 'line'],
+        'lines': [
+            ['write_waiters', 'write', 'incremental'],
+            ['read_waiters', 'read', 'incremental']
+        ]
+    },
+}
+
+# Stuff to gather - make tuples of DN dn and attrib to get
+SEARCH_LIST = {
+    'total_connections': (
+        'cn=Total,cn=Connections,cn=Monitor', 'monitorCounter',
+    ),
+    'bytes_sent': (
+        'cn=Bytes,cn=Statistics,cn=Monitor', 'monitorCounter',
+    ),
+    'completed_operations': (
+        'cn=Operations,cn=Monitor', 'monitorOpCompleted',
+    ),
+    'initiated_operations': (
+        'cn=Operations,cn=Monitor', 'monitorOpInitiated',
+    ),
+    'referrals_sent': (
+        'cn=Referrals,cn=Statistics,cn=Monitor', 'monitorCounter',
+    ),
+    'entries_sent': (
+        'cn=Entries,cn=Statistics,cn=Monitor', 'monitorCounter',
+    ),
+    'bind_operations': (
+        'cn=Bind,cn=Operations,cn=Monitor', 'monitorOpCompleted',
+    ),
+    'unbind_operations': (
+        'cn=Unbind,cn=Operations,cn=Monitor', 'monitorOpCompleted',
+    ),
+    'add_operations': (
+        'cn=Add,cn=Operations,cn=Monitor', 'monitorOpInitiated',
+    ),
+    'delete_operations':  (
+        'cn=Delete,cn=Operations,cn=Monitor', 'monitorOpCompleted',
+    ),
+    'modify_operations': (
+        'cn=Modify,cn=Operations,cn=Monitor', 'monitorOpCompleted',
+    ),
+    'compare_operations': (
+        'cn=Compare,cn=Operations,cn=Monitor', 'monitorOpCompleted',
+    ),
+    'search_operations': (
+        'cn=Search,cn=Operations,cn=Monitor', 'monitorOpCompleted',
+    ),
+    'write_waiters': (
+        'cn=Write,cn=Waiters,cn=Monitor', 'monitorCounter',
+    ),
+    'read_waiters': (
+        'cn=Read,cn=Waiters,cn=Monitor', 'monitorCounter',
+    ),
+}
+
+
+class Service(SimpleService):
+    def __init__(self, configuration=None, name=None):
+        SimpleService.__init__(self, configuration=configuration, name=name)
+        self.order = ORDER
+        self.definitions = CHARTS
+
+        self.server = configuration.get('server')
+        self.port = configuration.get('port')
```

Comment:
please add `DEFAULT_SERVER` and `DEFAULT_PORT` (localhost and 389)

Reply:
please do

Reply:
done